packages/components/tests/styles-test.js

```diff
@@ -22,7 +22,7 @@ const files = glob.sync('**/*.scss', {
 const render = promisify(sass.render);
 
 describe('styles', () => {
-  jest.setTimeout(20000);
+  jest.setTimeout(40000);
```

Comment:
for reviewers: tests were failing bc it said it was timing out, so I updated it to have more time